### ISSUE TITLE
Fix heading levels on home page

### ISSUE THREAD

**Issue by amzoss** [visualizingthefuture/examples-repository#110]
For accessibility, headings should always start with h1 and should never skip a heading level. Also, should be used logically (h3 should be used if there are subcategories under h2, etc.)

Need to specify a title (h1) for the page, probably the "Welcome..." statement. If that doesn't feel right, can make a new title on the page. It is better not to change the header to h1, IMO. The header appears on every page, and it would be illogical/improper for every page to have the same title.

Remaining page headings, if all on "same level", should all be h2. Headings for the recent items could then be h3.

If the font sizes feel too big at that point, can adjust with styling. Bootstrap has a ".small" class that might help? Not sure if it's a fixed small size or relative.
